orphans/src/Orphans/OpenApi.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE FlexibleInstances  #-}
+{-# LANGUAGE OverloadedStrings  #-}
+{-# LANGUAGE StandaloneDeriving #-}
+module Orphans.OpenApi () where
```

Comment:
I'm not sure what the advantage of putting this in a package of its own is. Why not just put it in a module in the package where you use it, or `plutus-ledger`? We already have orphans in there. It also doesn't make much sense to me to have a generic `orphans` package, would we put all orphans for all packages in there?

Reply:
The idea was to unify the orpans that can be shared with multiple packages. For example it needs for `plutus-pab` and `plutus-contract`.

We could put orphans into `plutus-ledger` but would it make sense to add a dependecy just for orphans alone? Or maybe we can just define the `Orphan` module for each package independetly upon requirement, since now it only is needed for some base types (`aeson` or `freer`)?

Reply:
Well, if you follow my suggestion about not having `plutus-ledger-api` and `plutus-core` depend on `openapi`, then you'll need a bunch of orphan instances for the types from those packages too. `plutus-ledger` seems like the best place to put all that, since the others depend on it already. Or maybe `plutus-contract`, WDYT @j-mueller ?

Reply:
Yes I was going to comment on that too - `plutus-ledger` shoud be fine.